.github/ISSUE_TEMPLATE/refinement.md

```diff
@@ -0,0 +1,79 @@
+<!--
+Short description
+-->
+
+<!--
+## Current situation
+
+What is the current user-flow.
+Where fitting, include screenshots or videos of the current pages or user-flow.
+-->
+
+<!--
+## Problem
+
+What in the current situation is problematic.
+Refer to other issues where possible.
+-->
+
+<!--
+## Desired situation
+
+Where fitting, include screenshots with annotations and notes.
+-->
+
+<!--
+## Solution
+
+What steps are involved to get from the current situation to the desired situation.
+This implies that taking these steps solves the problem.
+
+If this includes bigger sections of work, add subsections here.
+Attempt to describe these steps as a task-list.
+Attempt to estimate how much time it will take to complete each of these steps.
+
+Do the steps involved seem like too much work or hard to determine all at once? Scope the issue down or move the first step to a refinement sub-issue and continue from there.
+-->
+
+## Refine this issue
+
+- [ ] Give short description of this issue.
+- [ ] Determine with whom to discuss this with: @...
+
+    > This should start with a product owner.
+    > When it's a functional change, include Pieter van Tiel.
+    > When it's a technical change, include Pieter Bos.
```

Comment:
Pieter vt isn't a product owner anymore. I do believe he is essential in refining functional changes, but maybe in the `discuss the issue` part instead of the product owner part?

Reply:
Ah, I wasn't aware that was already the case. So Pieter Bos is the sole PO?
So Pieter Bos should refer to the right person to include in the process?

Reply:
Ah yeah ofc you missed the standup, forgot. Pieter Bos will be the sole PO until further notice (we are looking for a different PO).
Not saying he is the right person, but he is de PO. Good of you to check with them who wants to be notified :)